DigitalLearningSolutions.Web/Views/Register/Confirmation.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Register
+@model ConfirmationViewModel
+

```

Comment:
Do we need to set the title here so a tab has a correct title? Maybe "Registration Confirmation"?